CBLClient/Apps/CBLTestServer-Dotnet/TestServer.iOS/TestServer.iOS.csproj

```diff
@@ -22,6 +22,8 @@
     <MtouchArch>x86_64</MtouchArch>
     <MtouchLink>SdkOnly</MtouchLink>
     <MtouchDebug>true</MtouchDebug>
+    <CodesignProvision>VS: WildCard Development</CodesignProvision>
+    <CodesignKey>Apple Development: James Borden (8538L9XTA3)</CodesignKey>
```

Comment:
can you replace with mobille-team apple account? we need to make sure it works with mobile-team apple account